### PR TITLE
Added Eulerbeats events

### PR DESCRIPTION
**Pull request by nothingnix**
Used the Contract Parser: https://contract-parser.d5.ai

I parsed the Eulerbeats contract as an example ERC1155.

### CODE REVIEW

File: dags/resources/stages/parse/table_definitions/eulerbeats/EulerBeats_event_ApprovalForAll.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                "name": "operator",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "approved",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "Emitted when `account` grants or revokes permission to `operator` to transfer their tokens, according to
```
Comment (medvedev1088) [inline dags/resources/stages/parse/table_definitions/eulerbeats/EulerBeats_event_ApprovalForAll.json:51]: Unnecessary line break

Comment (nothingnix) [reply dags/resources/stages/parse/table_definitions/eulerbeats/EulerBeats_event_ApprovalForAll.json]: right, missed that one. fixed!

**Comment by medvedev1088** [comment]
There is also a way to create tables that includes events from all contracts. Here is an example of TransferSingle and TransferBatch https://github.com/blockchain-etl/ethereum-etl-airflow/pull/306

```
SELECT * 
FROM `blockchain-etl.ethereum_common.All_event_TransferBatch`
```

**Comment by nothingnix** [comment]
> There is also a way to create tables that includes events from all contracts. Here is an example of TransferSingle and TransferBatch #306 (notice contract_address is null)
> 
> ```
> SELECT * 
> FROM `blockchain-etl.ethereum_common.All_event_TransferBatch`
> ```

This is really helpful, thank you!

**Pull request merged**